scripts/album.js

```diff
@@ -191,12 +193,13 @@ var setupSeekBars = function() {
 };
 
```

Comment:
Actually, I'm working on #3 of the assignment now and starting to following the feedback :) 
I'll wrap all of this up today and have it ready to review together during our call tomorrow.